html/index.html

```diff
@@ -9,38 +9,40 @@
       <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
       <link href="https://getbootstrap.com/docs/5.3/assets/css/docs.css" rel="stylesheet">
 
-    <script src="login.js" defer type="module"></script>
+    <script src="login.js"  type="module"></script>
 </head>
-<body class="p-3 m-4 border-0 bd-example bg-light">
-    <div class="container my-5">
-      <div class="row justify-content-center align-items-center">
+<body  class="bg-dark text-white d-flex justify-content-center align-items-center" style="height: 70vh">
+  <div class="container mb-30  ">
+    <div class="row justify-content-center align-items-center">
         <div class="col-md-6">
-          <div class="card">
-            <div class="card-header">
-              <h1>Login</h1>
-            </div>
-            <div class="card-body">
-              <form id="login_form">
-                <div class="mb-3">
-                  <label for="email" class="form-label">Email address</label>
-                  <input type="email" class="form-control" aria-describedby="emailHelp" id="email" placeholder="example@mail.com"/>
-                  <div id="emailHelp" class="form-text">We'll never share your email with anyone else.</div>
+            <div class="card bg-secondary text-white"> <!-- dark card background and white text for the card -->
+                <div class="card-header">
+                    <h1>Login</h1>
                 </div>
-                <div class="mb-3">
-                    <label for="password" class="form-label">Password</label>
-                    <input type="password" id="password" class="form-control" aria-describedby="passwordHelpBlock" autocomplete="false" placeholder="******"/>
-                    <div id="passwordHelpBlock" class="form-text">
-                      Your password must be 8-20 characters long, contain letters and numbers, and must not contain spaces, special characters, or emoji.
-                    </div>
+                <div class="card-body">
+                    <form id="login_form">
+                        <div class="mb-3">
+                            <label for="email" class="form-label">Email *</label>
+                            <input type="email" class="form-control" aria-describedby="emailHelp" id="email" placeholder="example@mail.com"/>
+                            <div id="emailHelp" class="form-text text-light"><small id="emailError" class="text-danger"></small></div> <!-- light color text for hints -->
```

Comment:
Teksten i rød er vanskelig å lese. bytte til svart. 
<small id="emailError" class="text-emphasis">

Reply:
Jeg tenkte rød siden det er en form for feilmelding, svart error føler jeg kan være litt forvirrende?

Reply:
grunn til at jeg ville endre rød er at det er vanskelig å lese